cg-spec/editors_draft.html

```diff
@@ -901,6 +1327,99 @@ <h2>Translate Basic Graph Patterns</h2>
 
     </section>
 
+    <section id="builtin-functions">
+      <h2>Function Definitions</h2>
+
+      <p>SPARQL introduces operators and functions that can be used in an expression of a FILTER clause, a BIND clause, or a SELECT clause (see <a data-cite="SPARQL11-QUERY#OperatorMapping">Section&nbsp;17.3</a> and <a data-cite="SPARQL11-QUERY#SparqlOps">Section&nbsp;17.4</a> in [[SPARQL11-QUERY]]). While these operators and functions are defined to operate on <a>RDF terms</a> and <a>query variables</a>, to use them also for SPARQL-star they have to be defined to operate on <a>RDF-star terms</a> and <a>query variables</a>. To this end, when using these operators and functions in the context of SPARQL-star, their definitions as given in <a data-cite="SPARQL11-QUERY#SparqlOps">Section&nbsp;17.4</a> of [[SPARQL11-QUERY]] are extended by assuming that any mention of <code>RDF term</code> as a data type for operands is understood to be the type of all <a>RDF-star terms</a>.</p>
```

Comment:
SPARQL-star introduces ...?

Reply:
The following substitution might sound better:

`s/to use them also for SPARQL-star they have to be/for SPARQL-star they/`

Reply:
> SPARQL-star introduces ...?

I think this first section is more generally about SPARQL, so this is fine as-is IMO. Agree with the substitution sounding better.

Reply:
> SPARQL-star introduces ...?

Not really. As also mentioned by @jeenbroekstra, this first part is indeed about SPARQL.

> The following substitution might sound better: [...]

Agreed, and fixed (commit https://github.com/w3c/rdf-star/pull/118/commits/3622b7b34769af582f61c951b5c3bc86f512914f)